operator/cmd/mesh/testdata/profile-dump/output/all_off.yaml

```diff
@@ -1,3 +1,7 @@
+Proceeding despite the following validation errors: 
```

Comment:
Not sure why this is here - left over debug?

Reply:
I am not clear either. There should be several yaml files in testdata/ to be updated by the tool.

Reply:
I think this existed because of the proto gen issue before, now if you run make gen again it should not contain this output